rocAL/rocAL/include/augmentations/effects_augmentations/node_snp_noise.h

```diff
@@ -32,13 +32,16 @@ class SnPNoiseNode : public Node
 public:
     SnPNoiseNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     SnPNoiseNode() = delete;
-    void init(float sdev);
-    void init(FloatParam *sdev);
+    void init(float noise_prob, float salt_prob, float noise_value, float salt_value, int seed);
```

Comment:
We need to change braces to google style for this class as well
Please check and change